tests/tests/test_models.py

```diff
@@ -1,6 +1,10 @@
 from datetime import timedelta
+from email.mime.image import MIMEImage
+from email.generator import _fmt
 from unittest.mock import patch
+from amqp import RecoverableConnectionError
```

Comment:
Django-yubin doesn't use ampq :)

Reply:
aargh some auto-import got me again, this was not intentional :) I'll remove it